### PR TITLE
feat(i18n): Chinese: webhook action setting

### PR DESCRIPTION
**Pull request by xuanxuan1231**
### Description

Chinese: webhook action setting

### CODE REVIEW

File: locales/settings/zh-CN.json
Comment (barnett617) [inline locales/settings/zh-CN.json:1]: maybe use Chinese word "网络钩子" like [wikipedia](https://zh.wikipedia.org/wiki/%E7%BD%91%E7%BB%9C%E9%92%A9%E5%AD%90)

File: locales/errors/zh-CN.json
Comment (barnett617) [inline locales/errors/zh-CN.json:1]: prefer to use a more friendly word

File: locales/settings/zh-CN.json
Comment (DIYgod) [reply locales/settings/zh-CN.json:1]: Oh no, I feel this is too strange, I might have to think for a long time to figure out what it really is.

Comment (xuanxuan1231) [reply locales/settings/zh-CN.json:1]: A chinese website called Gitee:
image

So can we use `Webhook` or `Webhooks`?

Comment (barnett617) [reply locales/settings/zh-CN.json:1]: Maybe just keep it "Webhooks" is good for now which is as same as it in en.json

File: locales/settings/zh-CN.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   "actions.action_card.to": "到",
   "actions.action_card.translate_into": "翻译成",
   "actions.action_card.value": "值",
+  "actions.action_card.webhooks": "Webhook",
```
Comment (DIYgod) [inline locales/settings/zh-CN.json:42]: It seems that an 's' is missing, it's a countable noun.

**Pull request merged**